src/Configurations/AuthenticationConfiguration.js

```diff
@@ -9,7 +9,7 @@ export default class AuthenticationConfiguration
         return url2;
     }
 
-    static SignUp()
+    static signUp()
```

Comment:
Function naming should be in proper format

Reply:
> Function naming should be in proper format

Function Name Changed...All are in one format

Reply:
Changes Accepted!!